pepper-apis/studybuilder-cli/studies/lms/emails/germline-consent-addendum-invitation.html

```diff
@@ -82,7 +82,7 @@
     </tr>
     <tr style="width: 540px;">
         <td style="padding: 50px 0 0 0;">
-            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear -ddp.participant.firstName-,</p>
+            <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Dear Participant,</p>
```

Comment:
I believe this is incorrect. I dont think these changes are needed. We need name substituted.

Reply:
Maybe I misunderstood the request, but the acceptance criteria of the ticket says: 

AC2: Adult email recipients are addressed as “Participant”; blue highlighted text is “your”`

And the docs attached to the ticket have the salutation line as `Dear Participant/Parent` highlighted in blue. So I figured this meant that was meant to be changed. I can check with Whitney.

Reply:
I believe its original document for reference. Yes Whitney can confirm it.